site/components/pages/Home/index.tsx

```diff
@@ -65,16 +68,19 @@ export const Home: NextPage<Props> = (props) => {
 
       <Section variant="gray" className={styles.heroSection}>
         <div className="hero_container">
-          <div className="hero_newsBanner">
-            <Text t="button" align="end">
-              <Trans>📰 Recent Post: </Trans>
-            </Text>
-            <Link href="/blog/a-new-face-for-klimadao">
-              <a>
-                <Trans>A New Face For KlimaDAO</Trans>
-              </a>
-            </Link>
-          </div>
+          {props.latestPost && (
```

Comment:
If we fail to fetch data from sanity, instead of raising an error as we do in other places, allow the query to fail silently and don't render the news banner. Any thoughts on this?